scrapy_splash/middleware.py

```diff
@@ -45,8 +46,13 @@ class SplashCookiesMiddleware(object):
     It should process requests before SplashMiddleware, and process responses
     after SplashMiddleware.
     """
-    def __init__(self):
+    def __init__(self, debug=False):
         self.jars = defaultdict(CookieJar)
+        self.debug = debug
+
+    @classmethod
+    def from_crawler(cls, crawler):
+        return cls(debug=crawler.settings.getbool('COOKIES_DEBUG'))
```

Comment:
Yes, I like it better, especially since there are other settings prefixed with ``SPLASH_``, thanks! I updated the PR.